### PR TITLE
Adding java code for GCD using Euclid division method

### LINKED ISSUE #1525: Gcd implementation in java

### PR DESCRIPTION
**Pull request by eKta37**
Fixes #1525

### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added a complete code for gcd of two numbers in java 

#### Languages Used:
- Java 

#### Files Added:
- gcd.java

Thanks!

### CODE REVIEW

File: Gcd/gcd.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/*
+ * This program is based on Euclid's method to find the GCD of two numbers
+ * 
+ */
+
+//importing files
+import java.util.Scanner;
+public class gcd {
+	
+	public static int gcd_func(int a, int b) {
```
Comment (SakshiShreya) [inline Gcd/gcd.java]: Use 4 spaces instead of 1 tab.

File: Gcd/gcd.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		Scanner s = new Scanner(System.in);
+		int num1 = s.nextInt();
+		int num2 = s.nextInt();
+		
+		//function calling and storing gcd of given numbers to variable result
+		int result = gcd_func(Math.max(num1, num2), Math.min(num1, num2));
+		System.out.println("The GCD of given two numbers is " + result);
+	}
+
+}
```
Comment (SakshiShreya) [inline Gcd/gcd.java]: Please add sample input and output.

**Comment by jainaman224** [comment]
There is no activity on it since 2 days.

**Comment by eKta37** [comment]
I'll make a PR again today, sorry for being late
I am having my last exam today.

File: Gcd/gcd.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	public static int gcd_func(int a, int b) {
+		//base case
+		if(b == 0) {
+			return a;
+		}
+		return gcd_func(b, a%b);
+	}
+
+	public static void main(String[] args) {
+		//taking input
```
Comment (jainaman224) [inline Gcd/gcd.java]: Use 4 space indentation.

File: Gcd/gcd.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/*
+ * This program is based on Euclid's method to find the GCD of two numbers
+ * 
+ */
+
+//importing files
+import java.util.Scanner;
+public class gcd {
```
Comment (jainaman224) [inline Gcd/gcd.java]: Add a new line above this.

**Comment by somya-kapoor** [comment]
@eKta37 Are you working on it?

**Comment by eKta37** [comment]
Yes, I'm working on it

**Comment by somya-kapoor** [comment]
@eKta37 Please use 4 space indentation, you are using 8 space indentation.

**Comment by eKta37** [comment]
@jainaman224 @SakshiShreya please review it.

**Comment by somya-kapoor** [comment]
@eKta37 Please use 4 space indentation, you are using 8 space indentation.

**Comment by somya-kapoor** [comment]
@eKta37 Are you working on it?

**Comment by somya-kapoor** [comment]
@eKta37  Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**